javascripts/govuk/analytics/govuk-tracker.js

```diff
@@ -0,0 +1,128 @@
+;(function (global) {
+  'use strict'
+
+  var $ = global.jQuery
+  var GOVUK = global.GOVUK || {}
+
+  var GOVUKTracker = function (gifUrl, fieldsObject) {
```

Comment:
`fieldsObject` doesn't get used anywhere. If for the first pass of this we're not passing it through, should chop it out entirely.